bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "component-storage-selector",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
@donnapep These changes look to be a minor change that involve a new feature, version should be `1.1.0`.